Assignment2/TuxToy-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,41 @@
+{
+  "@context": {
+    "student": "http://www.tarea2b.es/student",
+    "full_name":"student:full_name",
+    "hometown":"student:hometown",
+    "age":"student:age",
+    "Alma_mater":"student:Alma_mater",
+    "parents":"student:parents",
+    "mother":"parents:mother",
+    "father":"parents:father",
```

Comment:
These attribute definitions work, but semantically they aren't ok. The best way is to use a prefix and add the name of the concept/property you want to use e.g. "mother":"yourvocab:mother". 
You don't need to change them, but remember to provide good definitions for the future.